lib/twitter.js

```diff
@@ -161,7 +161,7 @@ Twitter.prototype.post = function(url, content, content_type, callback) {
     }
     else {
       try {
-        var json = JSON.parse(data || {});
+        var json = data ? JSON.parse(data) : data;
         callback(null, json);
```

Comment:
I don't know, it would depend on what relies on this... if the original intention was to return `{}` (although it was wrongly coded) then it could be possible that somewhere, there's someone expecting and object and not null? Just guessing... If I'm right, all methods posting something will use this method right?